modules/luci-base/po/zh-cn/base.po

```diff
@@ -3921,16 +3930,16 @@ msgid "server mode"
 msgstr "服务器模式"
 
 msgid "stateful-only"
-msgstr "有状态的"
+msgstr "有状态"
 
 msgid "stateless"
-msgstr "无状态的"
+msgstr "无状态"
 
 msgid "stateless + stateful"
-msgstr "无状态的 + 有状态的"
+msgstr "无状态 + 有状态"
 
 msgid "tagged"
-msgstr "已标记"
+msgstr "有 VLAN 标记"
```

Comment:
The added "VLAN" in the translation is not needed.  It's clear from the context it's about VLAN tag.

Reply:
maybe, but I also observer that many people are confused by this, since they are not VLAN professors.

Reply:
Still they will not in better situation with the added VLAN text.  Please leave the original translation there, it describes the intention just fine.